cmd/node/main.go

```diff
@@ -34,7 +34,7 @@ func main() { // run the app
 	// TODO: Move version, relay subcommands from node service and smeshing
 	// service subcommands to root command.
 
-	if len(os.Args) > 1 {
+	if len(os.Args) > 1 && !helpFlagSet(os.Args) {
```

Comment:
That's equivalent without needing a helper function:
```suggestion
	if len(os.Args) > 1 && !slices.Contains(os.Args, "--help") {
```

The `-h` probably needs same "treatment", doesn't it?

---

I have a feeling we misuse cobra in the first place, but IDK cobra well enough to tell how to fix it...

Reply:
Done.